src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -27,7 +27,7 @@ public JedisPool(String host, int port) {
 
   public JedisPool(final String host) {
     URI uri = URI.create(host);
```

Comment:
@HeartSaVioR we can call `this(new GenericObjectPoolConfig(), URI.create(host))` because call another constructor must be the first command inside the constructor.

We could change the entire constructor to delegate to `this(new GenericObjectPoolConfig(), URI.create(host))`, but doing this, if the URI is invalid we will thrown one exception instead of trying to build the `GenericObjectPoll` with host string. I think that's not the desired action.